[
  {
    "poster": {
      "poster_name": "Milos Miljkovic",
      "poster_handle": "miljkovicmisa"
    },
    "time_stamp": "2024-10-06 12:31:10",
    "edited": {
      "time_stamp": "2024-10-11 12:15:03"
    },
    "text": "Hello everyone, i'm getting the following error with Storybook on a clean install from the repo:\n<https://github.com/HubSpot/cms-react/tree/main/examples/graphql-storybook|https://github.com/HubSpot/cms-react/tree/main/examples/graphql-storybook>\n\nDid some digging but didn't find anything useful. Some have reported such issues with Storybook and vite in the below issues:\n<https://github.com/storybookjs/storybook/issues/18920|https://github.com/storybookjs/storybook/issues/18920>\n<https://github.com/storybookjs/builder-vite/issues/467|https://github.com/storybookjs/builder-vite/issues/467>\n\nMy node.js version is: v20.17.0\n\nBelow is the error:\n```ReferenceError: process is not defined\n    at assertEach```"
  }
]